node/src/server-modules/GlideFt.ts

```diff
@@ -160,6 +162,31 @@ export class GlideFt {
             decoder: Decoder.String,
         }) as Promise<"OK">;
     }
+
+    /**
+     * Deletes an index and associated content. Indexed document keys are unaffected.
+     *
+     * @param client The client to execute the command.
+     * @param indexName The index name.
+     *
+     * @returns "OK"
+     *
+     * @example
+     * ```typescript
+     * // Example usage of FT.DROPINDEX to drop an index
+     * await GlideFt.dropindex(client, "json_idx1"); // "OK"
+     * ```
+     */
+    static async dropindex(
+        client: GlideClient | GlideClusterClient,
```

Comment:
maybe
```suggestion
        client: BaseClient,
```

Reply:
Then we need to export BaseClient to the user.  They don't need it.